examples/unimp_arxiv.py

```diff
@@ -0,0 +1,170 @@
+import logging
+import os
+
+import torch
+from ogb.nodeproppred import PygNodePropPredDataset
+
+import torch_geometric.transforms as T
+import torch_geometric.utils.mask as mask_util
+from torch_geometric.nn import TransformerConv
+
+logger = logging.getLogger(__file__)
+logging.basicConfig(level=logging.INFO)
+
+# Get dataset
+root = os.path.join("..", "data", "OGB")
+dataset = PygNodePropPredDataset(
+    "ogbn-arxiv",
+    root,
+    transform=T.Compose([T.ToUndirected()]),
+)
+data = dataset[0]
+num_classes = 40
+
+# Form masks for labels: mask gives labels being predicted
+split_idx = dataset.get_idx_split()
+train_mask = mask_util.index_to_mask(split_idx["train"], size=data.num_nodes)
+valid_mask = mask_util.index_to_mask(split_idx["valid"], size=data.num_nodes)
+test_mask = mask_util.index_to_mask(split_idx["test"], size=data.num_nodes)
+
+# Model parameters
+inner_dim = 16
+heads = 2
+
+
+def ratio_mask(mask: torch.Tensor, ratio: float, shuffle: bool = False):
```

Comment:
I like it. How about we add `ratio_mask` as a (static) method to `MaskLabel`?

Reply:
💯